spec/lib/tasks/feature_flags_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rake'
+
+RSpec.describe 'Feature Flags', :rake do
+  before(:all) do
+    task_defined = Rake::Task.task_defined?('feature_flags:initialize')
```

Comment:
why wouldn't it be defined?

Reply:
@santib I'm not sure but when I run the test without this `before` block I get the following error:
![Screen Shot 2023-08-18 at 14 04 42](https://github.com/rootstrap/rails_api_base/assets/35469032/a8368efe-d2ec-49ca-8994-c443f6de7167)

Also, I found it to be common practice to do this in these kinds of tests.